### ISSUE TITLE
Start Screen: Application Background

### ISSUE THREAD

**Issue by mtg1986** [IBVSoftware/CollaBoard-Feedback#470]
When I change the application background nothing happens. 

Usability is not that nice as the DropDown menu does not disappear when I select a Color. After selecting a Color, nothing happens. 
Is this not a setting that needs to be done on a project and not on an application level?
Do we need that in the start screen when it is on project level?

![image](https://user-images.githubusercontent.com/19570958/35328360-59afe2d2-00fc-11e8-9b77-aa47281dd1aa.png)

![desktop background](https://user-images.githubusercontent.com/19570958/35328447-aa684b88-00fc-11e8-8d79-4a166e4df3d3.gif)

**Comment by corradocavalli** [comment]
Background color is not a project setting is application, so it is ok to have it. 
Ok, to make it close on selection.

**Comment by mtg1986** [comment]
In general my question would be if we need settings like we have here on the Start screen. If a users changes here something nothings visual happens. He does not see the minimap or the QAT. Only setting that is relevant here is the language, right?

**Comment by mtg1986** [comment]
If I look on the thumbnails of the projects, then the background Color is project dependent. For me this would make more sense. 

![image](https://user-images.githubusercontent.com/19570958/35332412-253af28a-010b-11e8-9da0-212da5a5bc06.png)

**Comment by corradocavalli** [comment]
Are you proposing to remove settings from Start screen?

**Comment by mtg1986** [comment]
Yes, because when the user is doing something nothing happens, so we might confuse them. 
The only setting that is relevant here is the language from my perspective. But before chaning this, we need to sync with Gian.

**Comment by gsantopaolo** [comment]
At the moment I don't have an answer need to discuss

**Comment by gsantopaolo** [comment]
@mtg1986  need to discuss before the release...

**Comment by mtg1986** [comment]
I add this to the agenda of the next meeting so we can discuss.
